### PR TITLE
Jordan/refactor lazy send

### LINKED ISSUE #238: Refactor LazySendModal to be global for all balances
Every `BalanceRow` shouldn't need to import a `LazySendModal`, since there is only one open at a time.

Therefore we should move the `SendModal` component  to the `Balances` component and handle a click on a send button to trigger an event that bubbles to the balances component and opens the modal.

### PR DESCRIPTION
**Pull request by jbibla**
closes #238

### CODE REVIEW

File: components/portfolio/BalanceRow.vue [hunk 1]
```diff
@@ -128,24 +121,15 @@ export default {
       }
       let colour = '#'
       for (let i = 0; i < 3; i++) {
-        // eslint-disable-next-line prettier/prettier
-        const value = (hash >> (i * 8)) & 0xFF
+        // eslint-disable-next-line
+        const value = (hash >> (i * 8)) & 0xff
```
Comment (faboweb) [inline components/portfolio/BalanceRow.vue:125]: sure this works?

File: components/portfolio/Balances.vue [hunk 2]
```diff
@@ -23,17 +23,16 @@
         :balance="balance"
         :total-rewards-per-denom="totalRewardsPerDenom"
         :send="true"
+        @open-send-modal="openSendModal(balance.denom)"
       />
     </TableContainer>
-    <LazySendModal ref="SendModal" :denoms="getAllDenoms" />
+    <SendModal ref="SendModal" :denoms="getAllDenoms" />
```
Comment (faboweb) [inline components/portfolio/Balances.vue:29]: why not lazy`?

**Comment by Bitcoinera** [comment]
Tested this and it is working :+1:

File: components/portfolio/BalanceRow.vue (hunk 1 above)
Comment (jbibla) [reply components/portfolio/BalanceRow.vue:125]: just linting nonsense

i will revert and see if the linter doesn't complain

File: components/portfolio/Balances.vue (hunk 2 above)
Comment (jbibla) [reply components/portfolio/Balances.vue:29]: i thought it was interfering with events - but it wasn't - i will revert

File: components/portfolio/BalanceRow.vue (hunk 1 above)
Comment (jbibla) [reply components/portfolio/BalanceRow.vue:125]: my linter is forcing it to `0xff`

**Pull request merged**